main_bayesian.py

```diff
@@ -30,7 +30,7 @@ def getModel(net_type, inputs, outputs, layer_type, activation_type):
         raise ValueError('Network should be either [LeNet / AlexNet / 3Conv3FC')
 
 
-def train_model(net, optimizer, criterion, trainloader, num_ens=1, beta_type=0.1):
+def train_model(net, optimizer, criterion, trainloader, num_ens=1, beta_type=0.1, epoch=1, num_epochs=1):
```

Comment:
```suggestion
def train_model(net, optimizer, criterion, trainloader, num_ens=1, beta_type=0.1, epoch=None, num_epochs=None):
```
Instead of setting `beta=1` (which results in underfitting of the model), I would prefer if an error in raised in `metrics.get_beta` method.